core/templates/pages/skill-editor-page/editor-tab/skill-description-editor/skill-description-editor.directive.html

```diff
@@ -9,6 +9,11 @@
         <input type="text" class="form-control protractor-test-skill-description-field" ng-if="canEditSkillDescription()"
                ng-model="tmpSkillDescription" ng-blur="saveSkillDescription(tmpSkillDescription)"
                maxlength="64" ng-trim="false">
+        <span class="oppia-input-box-subtitle">
+          <i>
+            Skill description should be atmost 64 characters.
```

Comment:
Here and elsewhere: should be "at most".

Also, here and elsewhere, don't hardcode the 64. Interpolate it from the correct constant. It should be possible to change that constant in the future and have everything update at once.

Reply:
Done